docs/source/Quickstart.rst

```diff
@@ -314,3 +314,18 @@ Extracts a constraint from a cube. Time constraints work irrespective of whether
    >>> extracted_cube = ch.extract(cube, constraint)
    >>> extracted_cube
    <iris 'Cube' of air_temperature / (K) (time: 146; latitude: 145; longitude: 192)>
+
+Fixing known issues
+^^^^^^^^^^^^^^^^^^^
+There are some errors in CMIP data that are known about and can be fixed automatically. For details on the errors
+that can currently be fixed please see the :py:func:`cube_helper.fix_known_issues` documentation.
+
+.. code-block:: python
+
+   >>> cube = ch.load('/path/to/data/CMIP6/CMIP/CAS/FGOALS-f3-L/historical/r1i1p1f1/day/psl/gr/v20191019/')
```

Comment:
Should there be an `import cube_helper as ch` line before this one?

Reply:
The lack of import is consistent throughout the quickstart page; there is an import at the start of the page. It's also consistent with the Iris User Guide and similar documentation. I think that I will leave this as it is.